lib/opensearch/api/actions/cat/cluster_manager.rb

```diff
@@ -3,51 +3,47 @@
 # The OpenSearch Contributors require contributions made to
 # this file be licensed under the Apache-2.0 license or a
 # compatible open source license.
-#
-# Modifications Copyright OpenSearch Contributors. See
-# GitHub history for details.
+
+# This file is generated from the OpenSearch REST API spec.
+# Do not modify it by hand. Instead, modify the generator or the spec.
+
+# frozen_string_literal: true
 
 module OpenSearch
   module API
     module Cat
       module Actions
-        # Returns information about the cluster_manager node.
+        # Returns information about the cluster-manager node.
         #
-        # @option arguments [String] :format a short version of the Accept header, e.g. json, yaml
-        # @option arguments [Boolean] :local Return local information, do not retrieve the state from cluster_manager node (default: false)
-        # @option arguments [Time] :cluster_manager_timeout Explicit operation timeout for connection to cluster_manager node
-        # @option arguments [List] :h Comma-separated list of column names to display
-        # @option arguments [Boolean] :help Return help information
-        # @option arguments [List] :s Comma-separated list of column names or column aliases to sort by
-        # @option arguments [Boolean] :v Verbose mode. Display column headers
-        # @option arguments [Hash] :headers Custom HTTP headers
-        #
-        #
-        def cluster_manager(arguments = {})
-          headers = arguments.delete(:headers) || {}
-
-          arguments = arguments.clone
+        # @option args [String] :cluster_manager_timeout Operation timeout for connection to cluster-manager node.
+        # @option args [String] :format A short version of the Accept header (for example, `json`, `yaml`).
+        # @option args [Enumerable<String>] :h Comma-separated list of column names to display.
+        # @option args [Boolean] :help Return help information.
+        # @option args [Boolean] :local Return local information, do not retrieve the state from cluster-manager node.
+        # @option args [String] :master_timeout DEPRECATED Operation timeout for connection to cluster-manager node.
+        # @option args [Enumerable<String>] :s Comma-separated list of column names or column aliases to sort by.
+        # @option args [Boolean] :v Verbose mode. Display column headers.
+        def cluster_manager(args = {})
+          args = Utils.normalize_arguments(args)
+          headers = args.delete('headers') || {}
```

Comment:
Don't modify hash parameters, dup the arg first, https://code.dblock.org/2020/11/11/dup-dont-modify-input-hash-parameters-in-ruby.html.

Reply:
That's what `Utils.normalize_arguments(args)` does. It creates a new hash where all keys are converted to string and all URL params are URI encoded.